learn/polkadot/polkadot.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Tanssi and Polkadot
+description: Introduction to polkadot
+---
+
+# Tanssi and Polkadot
+
+Tanssi is a protocol built whithin the polkadot ecosystem, but what is Polkadot and why is it the right choice ?
```

Comment:
We don't typically start with text right underneath the h1 title so this does look a bit weird (cause im not used to it) haha so just raising this as something to consider